### PR TITLE
docs(ivy): improve debug documentation

### PR DESCRIPTION
**Pull request by ocombe**

### CODE REVIEW

**Review by mhevery — APPROVED** [review]

File: docs/BAZEL.md [hunk 1]
```diff
@@ -81,6 +81,9 @@ See also: [`//.bazelrc`](https://github.com/angular/angular/blob/master/.bazelrc
 
 The process should automatically connect to the debugger. For additional info and testing options, see the [nodejs_test documentation](https://bazelbuild.github.io/rules_nodejs/node/node.html#nodejs_test).
 
+- Click on "Resume script execution" to let the code run until the first `debugger` statement.
+- If you're debugging an ivy test and you want to see the template instructions, disable the `JavaScript source maps` in the settings, and look for `App_Template` in the call stack.
```
Comment (pkozlowski-opensource) [inline docs/BAZEL.md]: "see the template instructions" => "see/ inspect generated template instructions"?

Using exact `App_Template` name in a general instructions like this is misleading IMO, as the exact name in the call stack will depend on a component name - so we should have something like: `[class name]_Template`, ex.: `App_Template`. Would be good to note that hitting a stack-trace is not the only place where we can get access to the generated code - one can also go to sources and look into `ng://` namespace to see all the generated code.

Other remark: disabling source maps is not necessary, AFAIK, you can just click in a right-bottom corner or a source file view to display original  version.

File: docs/BAZEL.md [hunk 2]
```diff
@@ -81,6 +81,9 @@ See also: [`//.bazelrc`](https://github.com/angular/angular/blob/master/.bazelrc
 
 The process should automatically connect to the debugger. For additional info and testing options, see the [nodejs_test documentation](https://bazelbuild.github.io/rules_nodejs/node/node.html#nodejs_test).
 
+- Click on "Resume script execution" to let the code run until the first `debugger` statement.
```
Comment (pkozlowski-opensource) [inline docs/BAZEL.md]: "first `debugger` statement" => "first `debugger` statement or a previously set breakpoint"?

**Review by pkozlowski-opensource — CHANGES_REQUESTED** [review]
@ocombe thnx for adding those instructions, we should be really doing better in documenting our internal workflows so this PR improves the situation.

Having said this I find some parts of this PR slightly misleading (or not generic enough) - see the the in-line comments. Can you think of making those instructions more generic? Thnx!

File: docs/BAZEL.md (hunk 1 above)
Comment (ocombe) [reply docs/BAZEL.md]: That is great! I bet that even @kara didn't know that :) You can actually click on the link at the bottom of the code to see the js code.

**Review by pkozlowski-opensource — APPROVED** [review]

**Review by alexeagle — APPROVED** [review]

**Pull request closed without merging**